### ISSUE TITLE
Use sixels on Windows when available

### ISSUE THREAD

**Issue by johnd0e** [hpjansson/chafa#215]
Latest Windows Terminal supports sixels output, and it works when I specify `-f sixels` option explicitly

Perhaps it would be better to use it by default.

**Comment by hpjansson** [comment]
Yes - at least long term, that's how I'd like it. But we can't distinguish between older Windows Terminal and the newer versions with sixel support, so we should probably wait until more users have upgraded.

Do people generally keep Windows Terminal up to date?

**Comment by ykhan21** [comment]
@hpjansson, the yazi developer seems to query the terminal for sixels support like so: (https://github.com/microsoft/terminal/discussions/17809#discussioncomment-10469993). I don't know if this is feasible for chafa though.

**Comment by hpjansson** [comment]
> @hpjansson, the yazi developer seems to query the terminal for sixels support like so: ([microsoft/terminal#17809 (comment)](https://github.com/microsoft/terminal/discussions/17809#discussioncomment-10469993)). I don't know if this is feasible for chafa though.

It is, I just have to get over my API anxiety and implement an async terminal handler.

**Comment by a-usr** [comment]
You could use the ""Send Device Attributes" escape sequence to detect sixel support.

**Comment by hpjansson** [comment]
Yep. Similar discussion in #191.
